02-how-to-use.Rmd

```diff
@@ -2,179 +2,95 @@
 
 **Abstract**
 
-This chapter describes different ways the reader can use these book to learn about using R and data science tools in their education job. Job descriptions, lifestyles, and programming experience differ for everyone. Learning how to program in R on the job or at home will also look different to each reader. Applying R and data science tools in an education job requires learning these skills in a practical and meaningful context. The chapter describes three suggested ways to learn from the book, taking the reader’s experience into account. It also introduces the reader to ways they can support and contribute to the book’s content. This reinforces the theme of building content based on stories from the data science in education community.
+This chapter describes different ways the reader can use these book to learn about using R and data science tools in their education job. Job descriptions, lifestyles, and programming experience differ for everyone. Learning how to program in R on the job or at home will also look different to each reader. Applying R and data science tools in an education job requires learning these skills in a practical and meaningful context. The chapter describes three suggested ways to learn from the book, so that the reading experience can be customized to different learning journeys. It also introduces the reader to ways they can support and contribute to the book’s content. 
```

Comment:
Minor, change "the reader" to "you" to be more personal?

Reply:
And change "data science tools" to "new data sources and data analysis techniques"?

Reply:
@joshuarosenberg Since this is the abstract, I think it's meant to be "the reader".

To replace "data science tools", which mention of them do you mean?

Reply:
good with "the reader"!

and how about changing the first mention, and keeping the second?